### ISSUE TITLE
Processes many files in node_modules

### ISSUE THREAD

**Issue by ef4** [ember-codemods/ember-data-codemod#54]
This is followup to https://github.com/ember-cli/ember-cli-update-codemods-manifest/issues/59.

Here are reproduction instructions.

1. Checkout https://github.com/ember-animation/ember-animated at commit 4792f0ab48d0397900fbad1252f7aa5f1d2b93cf
2. `yarn install`
3. `npx ember-data-codemod . "tests/dummy/app/**/*.js"`

What I see:

```
npx: installed 304 in 8.757s
Processing 19378 files...
Spawning 7 workers...
Sending 50 files to free worker...
Sending 50 files to free worker...
Sending 50 files to free worker...
Sending 50 files to free worker...
Sending 50 files to free worker...
Sending 50 files to free worker...
Sending 50 files to free worker...
Sending 50 files to free worker...
Sending 50 files to free worker...
Sending 50 files to free worker...
Sending 50 files to free worker...
 ERR node_modules/charenc/README.js Transformation error (Unexpected token (1:0))
SyntaxError: Unexpected token (1:0)
    at Object.raise (/Users/edward/.npm/_npx/15090/lib/node_modules/ember-data-codemod/node_modules/@babel/parser/src/parser/location.js:41:63)
    at Object.unexpected (/Users/edward/.npm/_npx/15090/lib/node_modules/ember-data-codemod/node_modules/@babel/parser/src/parser/util.js:160:16)
    at Object.parseExprAtom (/Users/edward/.npm/_npx/15090/lib/node_modules/ember-data-codemod/node_modules/@babel/parser/src/parser/expression.js:1117:20)
    at Object.parseExprAtom (/Users/edward/.npm/_npx/15090/lib/node_modules/ember-data-codemod/node_modules/@babel/parser/src/plugins/estree.js:279:24)
    at Object.parseExprAtom (/Users/edward/.npm/_npx/15090/lib/node_modules/ember-data-codemod/node_modules/@babel/parser/src/plugins/jsx/index.js:527:22)
    at Object.parseExprSubscripts (/Users/edward/.npm/_npx/15090/lib/node_modules/ember-data-codemod/node_modules/@babel/parser/src/parser/expression.js:527:23)
    at Object.parseMaybeUnary (/Users/edward/.npm/_npx/15090/lib/node_modules/ember-data-codemod/node_modules/@babel/parser/src/parser/expression.js:507:21)
    at Object.parseExprOps (/Users/edward/.npm/_npx/15090/lib/node_modules/ember-data-codemod/node_modules/@babel/parser/src/parser/expression.js:275:23)
    at Object.parseMaybeConditional (/Users/edward/.npm/_npx/15090/lib/node_modules/ember-data-codemod/node_modules/@babel/parser/src/parser/expression.js:230:23)
    at Object.parseMaybeAssign (/Users/edward/.npm/_npx/15090/lib/node_modules/ember-data-codemod/node_modules/@babel/parser/src/parser/expression.js:180:21)
Sending 50 files to free worker...
Sending 50 files to free worker...
Sending 50 files to free worker...
Sending 50 files to free worker...
Sending 50 files to free worker...
Sending 50 files to free worker...
Sending 50 files to free worker...
Sending 50 files to free worker...
Sending 50 files to free worker...
```

There are not 19,000 files in the actual project. And the first reported error is for a file inside node_modules that should not have been processed. This leads me to believe that the codemod is incorrectly processing all files under node_modules.

**Comment by makepanic** [comment]
This seems to happen because the released `ember-data-codemod` package is outdated.

Try `npx github:ember-codemods/ember-data-codemod . path/to/models` instead

**Comment by dcyriller** [comment]
Thank you for the detailed reproduction @ef4.

This is true @makepanic. I just released v0.1.0 to fix that.
